docs/README.md

```diff
@@ -51,12 +51,18 @@ contribute to this project.
 
 ## Quickstart
 
-Anchor Platform can be run locally using Docker Compose. This will start an instance of the Anchor Platform and the
-Kotlin reference server.
+Anchor Platform can be run locally using Docker Compose. 
 
+### For version 2.x.x stable release
 ```shell
-docker build --build-arg BASE_IMAGE=gradle:7.6.4-jdk17 -t anchor-platform:local ./
-docker compose -f service-runner/src/main/resources/docker-compose.yaml up -d
```

Comment:
Should we provide a way for developers to start the stack against a locally built image?